zkclient/zkclient.conf.go

```diff
@@ -37,12 +37,12 @@ debug = no
 # requires debug = yes
 profiler = 127.0.0.1:6061
 
-# Valid ui colors: black, red, green, yellow, blue, magenta, cyan and white
+# Valid ui colors: na, black, red, green, yellow, blue, magenta, cyan and white
```

Comment:
I ended up copying the name of the ttk constant here but we could use "default" instead.